MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -193,10 +195,20 @@ extension MMEEventsManager {
         return eventDictionary
     }
     
-    func navigationCancelEvent(routeController: RouteController) -> [String: Any] {
+    //TODO: Change event semantic to `.exit`
+    func navigationCancelEvent(routeController: RouteController, rating: Int = unrated, comment: String? = nil) -> [String: Any] {
         var eventDictionary = self.addDefaultEvents(routeController: routeController)
         eventDictionary["event"] = MMEEventTypeNavigationCancel
         eventDictionary["arrivalTimestamp"] = routeController.sessionState.arrivalTimestamp?.ISO8601 ?? NSNull()
+        
+        let validRating: Bool = (rating >= MMEEventsManager.unrated && rating <= 100)
```

Comment:
Is it possible for us to ever set a non valid rating?

Reply:
Us? Not currently. Others? Yes. I mean `RouteMapViewController` could be subclassed… also `RouteController.sendCancelEvent(rating:comment:)` is `open` so anyone could call it (which may be undesirable)